samples/tfm/tfm_psa_template/Kconfig.sysbuild

```diff
@@ -0,0 +1,30 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+source "${ZEPHYR_BASE}/share/sysbuild/Kconfig"
+
+if BOARD_NRF5340DK_NRF5340_CPUAPP_NS
+
+choice NETCORE
+	default NETCORE_EMPTY
+endchoice
+
+config SECURE_BOOT_NETCORE
+	default y
+
+config NETCORE_APP_UPDATE
+	default y
+
+config MCUBOOT_APP_SYNC_UPDATEABLE_IMAGES
+	default y
+
+config PM_EXTERNAL_FLASH_MCUBOOT_SECONDARY
+	default y
+
+config MCUBOOT_USE_ALL_AVAILABLE_RAM
```

Comment:
Shouldn't the MCUBOOT_NRF_CLEANUP_NONSECURE_RAM be also y here?

Reply:
This is supposed to call this:
![image](https://github.com/user-attachments/assets/14e30f15-3a1d-4fde-8911-8b7aea3dd9ff)
before passing ctrl to application.

Reply:
Thank you for pointing out this configuration option and apologies for slow response.

It seems that the SB_CONFIG_MCUBOOT_USE_ALL_AVAILABLE_RAM that we are setting in here, already sets the
CONFIG_MCUBOOT_NRF_CLEANUP_NONSECURE_RAM for bootloader in:

https://github.com/nrfconnect/sdk-nrf/blob/main/sysbuild/CMakeLists.txt#L164-L167

So it should be unnecessary to add CONFIG_MCUBOOT_NRF_CLEANUP_NONSECURE_RAM for the bootloader.